cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/reserve_transfer.rs

```diff
@@ -1132,8 +1145,666 @@ fn reserve_transfer_native_asset_from_para_to_para_through_relay() {
 		<ForeignAssets as Inspect<_>>::balance(relay_native_asset_location, &receiver)
 	});
 
-	// Sender's balance is reduced by amount sent plus delivery fees
+	// Sender's balance is reduced by amount sent plus delivery fees.
 	assert!(sender_assets_after < sender_assets_before - amount_to_send);
+	// Receiver's balance is increased by `amount_to_send` minus delivery fees.
+	assert!(receiver_assets_after > receiver_assets_before);
+	assert!(receiver_assets_after < receiver_assets_before + amount_to_send);
+}
+
+// =========================================================================
+// ======= Reserve Transfers - Sufficient Asset - AssetHub->Parachain ======
+// =========================================================================
+/// Reserve Transfers of a sufficient asset from System Parachain to Parachain should
+/// work without using any other token.
+#[test]
+fn reserve_transfer_sufficient_assets_from_system_para_to_para() {
+	use penpal_runtime::xcm_config::ASSET_HUB_ID;
+
+	let destination = AssetHubWestend::sibling_location_of(PenpalA::para_id());
+	let sov_penpal_on_ahr = AssetHubWestend::sovereign_account_id_of(destination.clone());
+	let sender = AssetHubWestendSender::get();
+	let fee_amount_to_send = ASSET_HUB_WESTEND_ED * 100;
+	let asset_amount_to_send = ASSET_HUB_WESTEND_ED * 100;
+	let asset_owner = AssetHubWestendAssetOwner::get();
+	let receiver = PenpalAReceiver::get();
+
+	// Create sufficient asset.
+	let asset_id = 1984;
+	AssetHubWestend::force_create_asset(
+		asset_id,
+		asset_owner,
+		true, // Mark it as sufficient.
+		70_000,
+		vec![(sender.clone(), ASSET_HUB_WESTEND_ED * 10000)],
+	);
+	let system_para_foreign_asset_location = Location::new(
+		1,
+		[Parachain(ASSET_HUB_ID), PalletInstance(ASSETS_PALLET_ID), GeneralIndex(asset_id.into())],
+	);
```

Comment:
You should just register USDT in genesis [like it is done for AH Rococo](https://github.com/paritytech/polkadot-sdk/blob/18ed309a37036db8429665f1e91fb24ab312e646/cumulus/parachains/integration-tests/emulated/chains/parachains/assets/asset-hub-rococo/src/genesis.rs#L73).

Then ready for use in tests

```suggestion
	let usdt_location = Location::new(
		1,
		[Parachain(ASSET_HUB_ID), PalletInstance(ASSETS_PALLET_ID), GeneralIndex(USDT_ID)],
	);
```